src/pages/index.js

```diff
@@ -1,22 +1,110 @@
-import React from "react"
-import { Link } from "gatsby"
+import React from "react";
 
-import Layout from "../components/layout"
-import Image from "../components/image"
-import SEO from "../components/seo"
+import Datas from '../Datas.json';
+// import { MDBCol, MDBContainer, MDBRow, MDBFooter } from "mdbreact";
+import { MDBCol, MDBContainer, MDBRow, MDBFooter } from "mdbreact";
+import '@fortawesome/fontawesome-free/css/all.min.css'; 
+import 'bootstrap-css-only/css/bootstrap.min.css'; 
+import 'mdbreact/dist/css/mdb.css';
+import "../css/index.css";
 
 const IndexPage = () => (
-  <Layout>
-    <SEO title="Home" />
-    <h1>Hi people</h1>
-    <p>Welcome to your new Gatsby site.</p>
-    <p>Now go build something great.</p>
-    <div style={{ maxWidth: `300px`, marginBottom: `1.45rem` }}>
-      <Image />
-    </div>
-    <Link to="/page-2/">Go to page 2</Link> <br />
-    <Link to="/using-typescript/">Go to "Using TypeScript"</Link>
-  </Layout>
+  <body>
+    <h1 className="heading">Maths Department Seminar</h1>
+    <div className="time">Feb 10-11.Online.Free</div>
+     <div className="desc">
+     <p>
+
+     Join us on March 2-3, 2021 for this free virtual event.
+      Connect with Gatsby community members from all over the world, 
+      advance your skills with free workshops, and hear the latest
+     announcements from the Gatsby team. 
+      Claim your spot!
+     </p>
+   </div>
+   {/* <div>
+     <Card/>
+   </div> */}
+    <section className="talks">
+      <div className="talk-head">
+        <h3>February 10,2021 </h3>
+        <h1 className="guest-lect">Guest Lectures</h1>
+        <p>Local Times Shown Below</p>
+      </div>
+     
+    </section>
+   
+    <ul className="guest-Lectures">
+    {Datas.guests.map((item, i) => (
+       <section className="talks">
+   <li key={i} >
```

Comment:
The key will be passed into the <section> tag. Keys are always passed into the topmost element that is to be returned.